packages/api/src/context.ts

```diff
@@ -58,6 +59,8 @@ export interface DataLoaderContext {
 
   readonly peerSchema: DataLoader<string, GraphQLSchema | null>
   readonly peerAdminSchema: DataLoader<string, GraphQLSchema | null>
+
```

Comment:
Idem

Reply:
they are there on purpose for better readability